content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,175 @@
+==================
+Add new applicants
+==================
+
+Once an applicant submits an application, either using the online application, or emailing a job
+position alias, an applicant card is automatically created in the *Recruitment* application.
+
+However, in some instances, applicants may need to be created manually in the database. This could
+be necessary if, for example, a company accepts paper applications in the mail, or is meeting
+prospective applicants at an in-person job fair.
+
+To view current applicants, navigate to the :menuselection:`Recruitment` app, then click the desired
+job position card. Doing so reveals the :guilabel:`Applications` page, which displays all
+applicants for that specific role, in a default Kanban view, organized by stage.
+
+Add new applicants from a job position's :guilabel:`Applications` page by using either: the
+:ref:`New <recruitment/create-new-applicant>` button or the :ref:`quick add
+<recruitment/quick-add-applicant>` button.
+
+.. _recruitment/create-new-applicant:
+
+New applicant form
+==================
+
+To add a new applicant, and include all the relevant information, begin by navigating to the main
+:menuselection:`Recruitment` app dashboard. Then, click on a job position card. Next, click the
+:guilabel:`New` button in the top-left corner of that job position's :guilabel:`Applications` page.
+
+The :guilabel:`Subject / Application` field is populated with the pre-selected job position, by
+default. Certain fields on the applicant card may also be pre-populated, depending on how the job
+position is configured. Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter`
+field, are pre-populated.
+
+Enter the following information on the new applicant form. Note that not all fields listed below may
+be visible.
```

Comment:
Since you have the following note detailing that some fields may not be present, I think you can remove this bit. Also since you've broken the applicant form instructions into sections, let's specify that in the instruction
```suggestion
Complete the fields in the following sections on the new applicant form.
```